src/traces/parcoords/plot.js

```diff
@@ -0,0 +1,51 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var parcoords = require('./parcoords');
+var Lib = require('../../lib');
+
+module.exports = function plot(gd, cdparcoords) {
+
+    var fullLayout = gd._fullLayout;
+    var svg = fullLayout._paper;
+    var root = fullLayout._paperdiv;
+    var data = cdparcoords.map(function(d, i) {
```

Comment:
Indeed, no need for the `Lib.extendDeep` - the only thing I still needed to retained is [the mapping](https://github.com/monfera/plotly.js/blob/parcoords-squashed-2/src/traces/parcoords/plot.js#L83) for the `[0]` indexed elements, which is still in `plot.js` as I didn't want to bury this middleware-like `map` into `parcoords.js`.